modules/services/window-managers/river.nix

```diff
@@ -100,10 +110,11 @@ in {
 
       extraCommands = lib.mkOption {
         type = types.listOf types.str;
-        default = [
+        default = if !cfg.systemd.runInService then [
           "systemctl --user stop river-session.target"
           "systemctl --user start river-session.target"
-        ];
+        ] else
+          [ ];
```

Comment:
You can use `lib.optionals (!cfg.systemd.runInService) [ ... ]` here.

Reply:
I don't understand why, but tests don't pass with this change. It passes if I leave the if/else tho.